### DISCUSSION TITLE
Match - Add possibility to add critical alert entitlement on provisioning profile creating

### DISCUSSION THREAD

**Discussion by sneps85** [fastlane/fastlane#20168] [General]
Please add the possibility to add the critical alert entitlement on provisioning profile creation. ATM it is not possible; at least i do not find anything on how to achieve this.

**Comment by lucgrabowski** [comment]
It is possible. If you were granted rights to use this entitlement by Apple Support, you can check how it's named in developer portal (when you edit a profile you have a dropdown with names of entitlements). You can use this name as "template_name" parameter for match. You can read about this parameter in https://docs.fastlane.tools/actions/match/
